assets/prototype/application/hooks/useZonedTime.ts

```diff
@@ -0,0 +1,21 @@
+import { zonedTimeToUtc, utcToZonedTime } from 'date-fns-tz';
+import useConfig from '../../application/services/config/useConfig';
+
+const useZonedTime = () => {
+	const { timezone } = useConfig();
+
+	const zonedTimeToUtcCb = (date: Date | string | number) => {
+		return zonedTimeToUtc(date, timezone);
+	};
+
+	const utcToZonedTimeCb = (isoDate: Date | string | number) => {
+		return utcToZonedTime(isoDate, timezone);
+	};
+
+	return {
+		zonedTimeToUtc: zonedTimeToUtcCb,
+		utcToZonedTime: utcToZonedTimeCb,
```

Comment:
CNO says:

> how about `localTimeToUtc` && `utcToLocalTime` instead of `zonedTimeToUtc` && `utcToZonedTime` ?
> then you also don't have to append the callbacks with `Cb` nor rename them within the returned object

Reply:
I don't know who is this CNO, but he has some great suggestions.
Updated.